test/performance/long-running-tests/README.md

```diff
@@ -0,0 +1,174 @@
+## Workload Benchmark for KubeStellar
+
+To generate the sample workload for KubeStellar performance experiments proceed as following:
+
+### Generate the workload traffic
+
+1. Clone the following clusterloader2 repo: 
+
+   ```bash 
+   git clone -b release-1.31 https://github.com/kubernetes/perf-tests.git
+   ```
+
+   Set the variable `CL2_DIR` to the path of the subdirectory `clusterloader2/` of the cloned repo. For example: 
+
+   ```bash 
+   export CL2_DIR=$HOME/perf-tests/clusterloader2
```

Comment:
The `export` makes the reader wonder if there is any hidden dependency on the environment variable named "CL2_DIR". Does any executable invoked in the instructions in this file depend on that environment variable? If so, this is not a great design choice, because hidden dependencies are not great. If not then I recommend removing the `export` and making the variable name lower case, to make it clear that this is NOT being exported.

Reply:
Yes, there is a small dependency. That env variable is used by a script under the /common directory. Hence, I added the following note at step-1 in the README file:
```
The script test/performance/common/setup-clusterloader2.sh uses the environment variable set above
```

This small dependency simplifies a lot the  `setup-clusterloader2.sh` and also that script contains a error checker for the env variable already.